modules/ossm-control-plane-deploy.adoc

```diff
@@ -92,9 +89,33 @@ $ oc login https://{HOSTNAME}:6443
 $ oc new-project istio-system
 ----
 
-. Create a `ServiceMeshControlPlane` file named `istio-installation.yaml` using the example found in "Customize the {ProductName} installation". You can customize the values as needed to match your use case.  For production deployments you _must_ change the default Jaeger template.
+. Create a `ServiceMeshControlPlane` file named `istio-installation.yaml` using the following example in "Customize the {ProductName} installation". For production deployments you _must_ change the default Jaeger template.
 
-. Run the following command to deploy the control plane:
+.Example version 2 istio-installation.yaml
```

Comment:
Missing the plus marks (+) around this example, so the next steps ends up restarting the numbering at 1 again in the output.